wordpress-mobile/gutenberg-mobile
# Fix for Placeholder that is missing on heading block

This PR does fix the Placeholder text that is missing on the heading block (and probably other non-paragraph text blocks too).

In this PR we're passing down the `blockType` prop to the native wrapper, and use it to wrap the text the first time the user inserts a new value in the field. In this way Aztec properly render the text on the screen and the GB toolbar updated accordingly.

<del>I still need to re-check the code that does adds the tags around the text, since i'm not fully convinced it's safe, but the general idea is here.</del>


GB side PR available here: https://github.com/WordPress/gutenberg/pull/14404

**To test this PR**

__Test 1__
- Add a new heading block
- The placeholder text should appear
- Start typing in the block
- The styling should be there and the toolbar updated

__Test 2__
- Remove all the text in a Heading block
- The placeholder text should appear again
- Start typing in the block
- The styling should be there and the toolbar updated

__Test 3__
- Add a new heading block
- The placeholder text should appear
- Paste a text in the block
- The styling should be there and the toolbar updated
